modules/core/src/main/java/org/apache/ignite/internal/processors/platform/cache/PlatformCache.java

```diff
@@ -1069,6 +1089,27 @@ private void loadCache0(BinaryRawReaderEx reader, boolean loc) {
         return super.processOutObject(type);
     }
 
+    /** {@inheritDoc} */
+    @Override public long processInStreamOutLong(int type, BinaryRawReaderEx reader) throws IgniteCheckedException {
```

Comment:
I think we can put `OP_SIZE_LONG` and `OP_SIZE_LONG_LOC` into `processInStreamOutLong` above as well, to avoid overriding one more method.

Reply:
Yes, you are right.